src/main/java/edu/usf/cutr/gtfsrtvalidator/validation/entity/TimestampValidation.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2011 Nipuna Gunathilake.
```

Comment:
Can you please update the copyright header to  "Copyright (C)  2011-2016  Nipuna Gun..., University of South Florida"?